operations/commit_queue.go

```diff
@@ -215,29 +217,57 @@ func listCommitQueue(ctx context.Context, client client.Communicator, ac *legacy
 }
 
 func listPRCommitQueueItem(ctx context.Context, item restModel.APICommitQueueItem, projectRef *model.ProjectRef, uiServerHost string) {
+	issue := restModel.FromAPIString(item.Issue)
+	displayPRAuthor(ctx, projectRef, issue)
+
 	prDisplay := `
            PR # : %s
             URL : %s
 `
-	prDisplayVersion := "          Build : %s/version/%s"
-
-	issue := restModel.FromAPIString(item.Issue)
 	url := fmt.Sprintf("https://github.com/%s/%s/pull/%s", projectRef.Owner, projectRef.Repo, issue)
 	grip.Infof(prDisplay, issue, url)
 
+	prDisplayVersion := "          Build : %s/version/%s"
 	if restModel.FromAPIString(item.Version) != "" {
 		grip.Infof(prDisplayVersion, uiServerHost, restModel.FromAPIString(item.Version))
 	}
+
 	grip.Info("\n")
 }
 
+// if we can display author, we do, but in the event of an error we skip it
+func displayPRAuthor(ctx context.Context, projectRef *model.ProjectRef, issue string) {
+	conf, err := evergreen.GetConfig()
+	if err != nil {
+		return
+	}
+	ghToken, err := conf.GetGithubOauthToken()
+	if err != nil {
+		return
+	}
+	prNum, err := strconv.Atoi(issue)
+	if err != nil {
+		return
+	}
+	pr, err := thirdparty.GetGithubPullRequest(ctx, ghToken, projectRef.Owner, projectRef.Repo, prNum)
+	if err != nil {
+		return
+	}
+	if pr != nil && pr.User != nil {
+		grip.Infof("Author of PR: %s", pr.User.GetLogin())
```

Comment:
would it work to lookup the Evergreen username with
 https://github.com/evergreen-ci/evergreen/blob/e70f04908d636a643a0f4b317c739953fc09f5de/model/user/db.go#L45

Reply:
I don't think you can because that would reequire a DB connection, and this is in the client.